queries.sql

```diff
@@ -56,3 +56,82 @@ SELECT owners.full_name, COUNT(animals.name) FROM owners JOIN animals ON owners.
 --  Bob             |     2
 --  Jennifer Orwell |     2
 --  Sam Smith       |     1
+
+SELECT animals.name, visits.visit_date FROM animals INNER JOIN visits ON animals.id = visits.animal_id WHERE visits.vet_id = 1 ORDER BY visits.visit_date DESC;
+--    name   | visit_date 
+-- ----------+------------
+--  Blossom  | 2021-01-11
+--  Plantmon | 2020-08-10
+--  Agumon   | 2020-05-24
+SELECT COUNT(animal_id) FROM visits WHERE vet_id = 3;
+--  count 
+-- -------
+--      4
+SELECT vets.name, specializations.species_id FROM vets LEFT JOIN specializations ON vets.id = specializations.vet_id;
+--        name       | species_id 
+-- ------------------+------------
+--  William Tatcher  |          1
+--  Stephanie Mendez |          2
+--  Stephanie Mendez |          1
+--  Jack Harkness    |          2
+--  Maisy Smith      |           
+SELECT animals.name, visits.visit_date FROM animals JOIN visits ON animals.id = visits.animal_id WHERE visits.vet_id = 3 AND visits.visit_date BETWEEN '2020-04-01' AND '2020-08-30';
+--   name   | visit_date 
+-- ---------+------------
+--  Agumon  | 2020-07-22
+--  Blossom | 2020-05-24
+SELECT animals.name, COUNT(visits.animal_id) FROM animals LEFT JOIN visits ON animals.id = visits.animal_id GROUP BY animals.name ORDER BY COUNT DESC;
+--     name    | count 
+-- ------------+-------
+--  Boarmon    |     4
+--  Plantmon   |     3
+--  Pikachu    |     3
+--  Blossom    |     2
+--  Agumon     |     2
+--  Angemon    |     2
+--  Devimon    |     1
+--  Charmander |     1
+--  Squirtle   |     1
+--  Gabumon    |     1
+--  Ditto      |     0
+SELECT animals.name, visits.visit_date FROM animals INNER JOIN visits ON animals.id = visits.animal_id WHERE visits.vet_id = 2 ORDER BY visits.visit_date ASC;
```

Comment:
- [ ] Also consider adding a `LIMIT` clause to this query to obtain a single result of who Maisy Smith's first visit was.